### DISCUSSION TITLE
Light/dark theme

### DISCUSSION THREAD

**Discussion by kendal-kirby** [twilio-labs/paste#2861] [General] [answered]
My team is working on a new internal product that will replace Monkey (One Admin). We will be utilizing Paste components and console patterns, but want to ensure the new tooling is not confused with Console. It should be very clear that someone is signed into One Admin and not the Twilio Console, but given our time constraints, we are unable to design an entirely new theme and have decided to use Paste dark theme in the meantime (Console is not ready for dark theme and it has not been prioritized). However, while looking through the documentation, we saw the following rule: "**Don’t use Dark theme as the only theme within your product.**" Can this rule ever be broken? Can you share some of the reasons we would not want to set dark theme as the only theme within the product? Thanks so much!

**Comment by serifluous** [comment, marked as answer]
Hey @kendal-kirby ! That guideline may be out-of-date from when Dark theme was still in an experimental phase. I'll get an update in to remove it—thanks for catching it/

Out of curiosity, were there other options you tried to differentiate One Admin and Console besides a new theme or swapping to dark theme?

**Reply by kendal-kirby** [reply]
Sounds great, thank you!

To answer your question about differentiating in other ways, there are some limitations. In order to speed up initial development, make it easy for future development by other collaborator teams, and allow for console configurations to map to the One Admin platform, One Admin and Console need to share common components, patterns, and layouts. This makes it pretty tough to change much of the look and feel. Small thing that could help is a One Admin logo in the Masthead. Would be interested to hear any ideas you may have though!

**Reply by serifluous** [reply]
(For the team to round out this conversation: We're following up in Slack/meetings. @libertycarlin and @kendal-kirby will be meeting into 2023 to collaborate)
